simulariumio/tests/converters/test_trajectory_converter.py

```diff
@@ -57,6 +58,12 @@ def mixed_agents_wrong_display_type2():
     return result
 
 
+def mixed_agents_invalid_agent_id():
+    result = mixed_agents()
+    result.agent_data.unique_ids[0][0] = MAX_AGENT_ID + 1
```

Comment:
maybe consider defining INVALID_AGENT_ID = MAX_AGENT_ID+1 up above, close to where MAX_AGENT is defined.

Reply:
oh, nevermind, I just realized this is in a unit test